Terminal.Gui/Application.cs

```diff
@@ -1556,7 +1556,7 @@ internal static void OnMouseEvent (MouseEvent mouseEvent)
             return;
         }
 
-        var view = View.FindDeepestView (Current, mouseEvent.X, mouseEvent.Y);
+        var view = View.FindDeepestView (Current, new (mouseEvent.X, mouseEvent.Y));
```

Comment:
Why new?

Better to make the MouseEventArgs itself have a point and pass that by ref.

Reply:
Can be done non-breaking by just adding the property and modifying the existing properties to update the underlying Point.

Reply:
Actually, thinking about it a bit more...

If the second parameter to FindDeepestView were `ref`, you can either supply one if you already have one or create it in the FindDeepestView method.

`out` also can do that, but it's a bit non-obvious and not recommended to consume an `out` param, even though it's legal.

And `ref` has the advantage it won't _let_ you create an ephemeral value at the callsite.

Lots of other options, too, but key is just trying to minimize how many get created and thrown away.